From 114f364cb5f63235a0aa2d91e10a716d26ba23b9 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 8 Nov 2016 13:47:45 -0800
Subject: [PATCH] debugger_posix.cc: execinfo.h is glibc/uclibc specific

uclibc also defines __GLIBC__ so it will work for both
however musl does not have this .h file and it will work
there too.

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 src/base/debug/debugger_posix.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/base/debug/debugger_posix.cc b/src/base/debug/debugger_posix.cc
index 8f35794..66924f9 100644
--- a/src/base/debug/debugger_posix.cc
+++ b/src/base/debug/debugger_posix.cc
@@ -17,7 +17,7 @@
 #include <string>
 #include <vector>
 
-#if !defined(OS_ANDROID) && !defined(OS_NACL)
+#if !defined(OS_ANDROID) && !defined(OS_NACL) && defined(__GLIBC__)
 #include <execinfo.h>
 #endif
 
-- 
2.10.2

